package.json

```diff
@@ -13,19 +13,24 @@
     "lint": "eslint .",
     "lint-watch": "watch 'yarn lint'",
     "test": "nyc mocha --require @babel/register --require spec/testSetup.js  spec/*.spec.js",
-    "tdd": "yarn test --watch"
+    "tdd": "yarn test --watch",
+    "exportAgent": "node ./src/utility/ExportAgent.js",
+    "importAgent": "node ./src/utility/ImportAgent.js"
```

Comment:
Have you tested these commands?

Reply:
I mean manually?

Reply:
And using capital letter filenames indiciates a class for me.
Maybe name it in camelCase.

Reply:
I'll rename that. I tested the commands a number of times manually but I would not add automatic tests for them.